src/screens/DetailGroupImage/index.js

```diff
@@ -52,8 +53,13 @@ const ShowDetailGroupImage = (props) => {
         data={images}
         initialScrollIndex={indexScroll}
         renderItem={({item, index}) => (
-          <TouchableOpacity key={index} onPress={() => openDetail(item.asset_url)}>
-            <ImageBackground resizeMode="cover" style={styles.image} source={{uri: item.asset_url}}>
+          <TouchableOpacity
+            key={index}
+            onPress={() => openDetail(item.asset_url)}>
+            <ImageBackground
+              resizeMode="cover"
+              style={styles.image}
+              source={{uri: item.asset_url}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `TouchableOpacity` component now includes a `key` prop directly. This is unnecessary as `key` is automatically assigned by the `FlatList` component when using `renderItem`. Remove the `key` prop from `TouchableOpacity`.
```diff
- <TouchableOpacity key={index} onPress={() => openDetail(item.asset_url)}>
+ <TouchableOpacity onPress={() => openDetail(item.asset_url)}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->